pkg/apis/acme/v1alpha2/types_challenge.go

```diff
@@ -24,6 +24,7 @@ import (
 
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:storageversion
```

Comment:
Needs to change when v1 is merged in
We never added this before but it breaks controller-gen when not in patching mode which is handy to run for like euh... generating a new version :D